### PR TITLE
[Brcm][SAIServerV2] Build SAI Serverv2 docker

### PR DESCRIPTION
**Pull request by richardyu-ms**
#### Why I did it
Support saiserver v2 with python3 and thrift 0.13.0

#### How I did it
add variables to support the saiserverv2
build different thrift in saithrift depends on saiserver version
build differernt versions of saiserver
make the saiserver and saiserver docker with version number

#### How to verify it
test done:
build two different versions of sasiserver in local build environment

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: platform/broadcom/libsaithrift-dev.mk [hunk 1]
```diff
@@ -2,19 +2,28 @@
...
 SAI_VER = 0.9.4
 
-LIBSAITHRIFT_DEV = libsaithrift-dev_$(SAI_VER)_amd64.deb
+LIBSAITHRIFT_DEV = libsaithrift$(SAITHRIFT_VER)-dev_$(SAI_VER)_amd64.deb
 $(LIBSAITHRIFT_DEV)_SRC_PATH = $(SRC_PATH)/sonic-sairedis/SAI
-$(LIBSAITHRIFT_DEV)_DEPENDS += $(LIBTHRIFT) $(LIBTHRIFT_DEV) $(PYTHON_THRIFT) $(THRIFT_COMPILER) $(BRCM_SAI) $(BRCM_SAI_DEV)
-$(LIBSAITHRIFT_DEV)_RDEPENDS += $(LIBTHRIFT) $(BRCM_SAI)
+#Support two different versions of thrift
+ifeq ($(SAITHRIFT_V2),y)
+$(LIBSAITHRIFT_DEV)_DEPENDS += $(LIBTHRIFT_0_13_0) $(LIBTHRIFT_DEV_0_13_0) $(PYTHON_THRIFT_0_13_0) $(PYTHON3_THRIFT_0_13_0) $(THRIFT_COMPILER_0_13_0)
```
Comment (RyoYang) [inline platform/broadcom/libsaithrift-dev.mk:9]: $(PYTHON_THRIFT_0_13_0) may need to be deleted after this [PR](https://github.com/Azure/sonic-buildimage/pull/9508) 's issue fixed.

**Comment by richardyu-ms** [comment]
for building saiserverv2, please refer to the change in https://github.com/opencomputeproject/SAI/pull/1388

File: platform/broadcom/libsaithrift-dev.mk (hunk 1 above)
Comment (richardyu-ms) [reply platform/broadcom/libsaithrift-dev.mk]: the related issue fixed in https://github.com/Azure/sonic-buildimage/pull/9688

**Review by RyoYang — APPROVED** [review]

**Review by xumia — APPROVED** [review]

**Comment by qiluo-msft** [comment]
This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

**Pull request merged**